adapters/v1/syft_test.go

```diff
@@ -0,0 +1,50 @@
+package v1
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/kinbiko/jsonassert"
+	"github.com/kubescape/kubevuln/core/domain"
+	"gotest.tools/v3/assert"
+)
+
+func fileContent(path string) []byte {
+	b, _ := os.ReadFile(path)
+	return b
+}
+
+func Test_syftAdapter_CreateSBOM(t *testing.T) {
+	tests := []struct {
+		name    string
+		imageID string
+		format  string
+		options domain.RegistryOptions
+		wantErr bool
+	}{
+		{
+			name:    "hello-world",
```

Comment:
suggestion: let’s use a more explicit name for the test case

The idea is the same as the one in a previous comment related to Grype.